src/Networking/NexusMods.Networking.NexusWebApi/ModUpdateService.cs

```diff
@@ -109,17 +115,17 @@ public async Task<PerFeedCacheUpdaterResult<PageMetadataMixin>> CheckAndUpdateMo
         // _lastUpdateCheckTime is updated.
         if (throttle)
         {
-            var timeLeft = UpdateCheckCooldownSeconds - (int)(DateTimeOffset.UtcNow - _lastUpdateCheckTime).TotalSeconds;
+            var timeLeft = UpdateCheckCooldownSeconds - (int)(_timeProvider.GetUtcNow() - _lastUpdateCheckTime).TotalSeconds;
             if (timeLeft > 0)
             {
                 _logger.LogInformation("Skipping update check due to rate limit ({cooldown} seconds). Time left: {timeLeft} seconds.", UpdateCheckCooldownSeconds, timeLeft);
-                return PerFeedCacheUpdaterResult<PageMetadataMixin>.WithStatus(CacheUpdaterResultStatus.RateLimited);
+                return PerFeedCacheUpdaterResult<PageMetadataMixin>.WithStatus(CacheUpdaterResultStatus.Throttled);
             }
-            _lastUpdateCheckTime = DateTimeOffset.UtcNow;
+            _lastUpdateCheckTime = _timeProvider.GetUtcNow();
         }
         else
         {
-            _lastUpdateCheckTime = DateTimeOffset.UtcNow;
+            _lastUpdateCheckTime = _timeProvider.GetUtcNow();
         }
```

Comment:
You can just set the lastUpdateTime outside the if, since the only case you don't want it has an early return

Reply:
Also update the text of the log to say throttling rather than rate limiting.

Reply:
Solved